LanguageFeatures/Wildcards/other_declarations_A01_t09.dart

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion Other declarations: top-level variables, top-level function
+/// names, type names, member names, etc. are unchanged. They can be named `_`
+/// as they are today.
+///
+/// @description Checks that an external top-level variable called `_` can be
+/// declared and accessed.
+/// @author sgrekhov22@gmail.com
+
+// SharedOptions=--enable-experiment=wildcard-variables
+
+import '../../Utils/expect.dart';
+
+external int _;
```

Comment:
This would be a compile-time error with 'dart compile js', right?

I would assume that we have a standard way to specify that a particular test should not be executed on some configurations, but I can't immediately see any examples (grepping for 'SharedOptions', 'requirements', 'web-numbers', and a few others).

Perhaps it's going to be an entry in co19/*.status?

Reply:
Yes, we are doing it in a `.status` files with a comment `SkipByDesign # Non-JS-interop external members are not supported`

Reply:
OK, thanks!